package.json

```diff
@@ -38,9 +41,9 @@
     "snyk": "^1.40.2"
   },
   "devDependencies": {
-    "istanbul": "^0.4.5",
-    "mocha": "3",
-    "supertest": "^2.0.0"
+    "istanbul-harmony": "0",
```

Comment:
`istanbul-harmony` was deprecated in favor of `istanbul@>=0.4.0`. Let's revert the first line of `devDependencies` please.

Reply:
Whoops, I didn't realize the new syntax is broken on that version of Istanbul. I would like to switch to their new CLI, https://github.com/istanbuljs/nyc, which I believe solves this problem. Either way I noticed this is fixed on `instanbul@v1.1.0-alpha.1` but I feel weird about using an alpha dependency.